book/src/http_consensus.md

```diff
@@ -0,0 +1,182 @@
+vote (that is why it is _effective_ `Gwei`).
+
+The following fields are returned:
+
+- `current_epoch_active_gwei`: the total staked gwei that was active (i.e.,
+	able to vote) during the current epoch.
+- `previous_epoch_active_gwei`: as above, but during the previous epoch.
+- `current_epoch_attesting_gwei`: the total staked gwei that had one or more
+    attestations included in a block during the current epoch (multiple
+	attestations by the same validator does not increase this figure).
+- `current_epoch_target_attesting_gwei`: the total staked gwei that attested to
+	the majority-elected Casper FFG target epoch during the current epoch. This
+	figure must be equal to or less than `current_epoch_attesting_gwei`.
+- `previous_epoch_attesting_gwei`: see `current_epoch_attesting_gwei`.
+- `previous_epoch_target_attesting_gwei`: see `current_epoch_target_attesting_gwei`.
+- `previous_epoch_head_attesting_gwei`: see `previous_epoch_head_attesting_gwei`.
+
+From this data you can calculate some interesting figures:
+
+#### Participation Rate
+
+`previous_epoch_attesting_gwei / previous_epoch_active_gwei`
+
+Expresses the ratio of validators that managed to have an attestation
+voting upon the previous epoch included in a block.
+
+#### Justification/Finalization Rate
+
+`previous_epoch_target_attesting_gwei / previous_epoch_active_gwei`
+
+When this value is greater than or equal to `2/3` it is possible that the
+beacon chain may justify and/or finalize the epoch.
+
+### HTTP Specification
+
+| Property | Specification |
+| --- |--- |
+Path | `/consensus/global_votes`
+Method | GET
+JSON Encoding | Object
+Query Parameters | `epoch`
+Typical Responses | 200
+
+### Parameters
+
+Requires the `epoch` (`Epoch`) query parameter to determine which epoch will be
+considered the current epoch.
+
+### Returns
+
+A report on global validator voting participation.
+
+### Example
+
+```json
+{
+    "current_epoch_active_gwei": 52377600000000,
+    "previous_epoch_active_gwei": 52377600000000,
+    "current_epoch_attesting_gwei": 50740900000000,
+    "current_epoch_target_attesting_gwei": 49526000000000,
+    "previous_epoch_attesting_gwei": 52377600000000,
+    "previous_epoch_target_attesting_gwei": 51063400000000,
+    "previous_epoch_head_attesting_gwei": 9248600000000
+}
+```
+
+## `/consensus/individual_votes`
+
+Returns a per-validator summary of how that validator performed during the
+current epoch.
+
+The [Global Votes](#consensusglobal_votes) endpoint is the summation of all of these
+individual values, please see it for definitions of terms like "current_epoch",
+"previous_epoch" and "target_attester".
+
+### HTTP Specification
+
+| Property | Specification |
+| --- |--- |
+Path | `/consensus/individual_votes`
+Method | GET
+JSON Encoding | Object
+Query Parameters | `epoch`
+Typical Responses | 200
+
+### Parameters
+
+Requires the `epoch` (`Epoch`) query parameter to determine which epoch will be
+considered the current epoch.
```

Comment:
I think this should be a `RequestBody` with `epoch` and `pubkeys` as parameters.